be/src/storage/tablet_manager.cpp

```diff
@@ -1403,9 +1409,15 @@ TabletManager::tablets_shard& TabletManager::_get_tablets_shard(TTabletId tablet
 }
 
 Status TabletManager::create_tablet_from_snapshot(DataDir* store, TTabletId tablet_id, SchemaHash schema_hash,
-                                                  const string& schema_hash_path) {
+                                                  const string& schema_hash_path, bool is_primary_key) {
```

Comment:
It seems that `is_primary_key` is always true.

Reply:
Can you please rename this method to something like `create_primary_key_tablet_from_snapshot`.